lib/engine/game/g_1880/step/check_fi_connection.rb

```diff
@@ -0,0 +1,128 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/base'
+
+module Engine
+  module Game
+    module G1880
+      module Step
+        class CheckFIConnection < Engine::Step::Base
+          ACTIONS = %w[destination_connection pass].freeze
+
+          def actions(entity)
+            return [] unless entity == current_entity
+            return [] unless entity.minor?
```

Comment:
If you are going to return no action unless a minor (which makes sense), then the auto action should do the same.